MaterialProgressBar/src/main/ets/components/basic/MaterialComponents.ets

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright (C) 2022 Application Library Engineering Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import CircularComponent from './CircularComponent'
+import LinearComponent from './LinearComponent'
+
+enum MaterialProgressStyle {
+  MaterialCircular,
+  MaterialLinear,
+}
+
+@Component
+struct MaterialComponent {
+  @State progress: number = 30
```

Comment:
Suggest to define a model / namespace and use it so that user also will be clear of the available parameters to be set

Reply:
The programmer would access the MaterialProgressBar.ets file where the model 'MaterialModel' would be built, the MaterialComponents.ets would be called. MaterialComponents.ets is not directly accessed by the programmer. This is where the user can check about the available parameters to be set.